README.md

```diff
@@ -14,6 +14,12 @@ terminal 1:
 minikube start --kubernetes-version=v1.25.3 --cpus max --memory max && skaffold run
 ```
 
+after the first time you have deployed the app you can run the following command to have faster deployments:
```

Comment:
Let's put this under Pro tip also. So our main is only "terminal 1" and "terminal 2"